### PR TITLE
Ids and some classNames to ease navigation in tests

### PR DESCRIPTION
**Pull request by marillam**
Added ids and classNames when there were many options without clear separating id.

Please comment if need for more naming

### CODE REVIEW

**Review by no-comma — APPROVED** [review]
LGTM

File: client/src/components/InstanceSummaryView.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ const InstanceSummaryView = () => {
       </Box>
       <Box sx={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', alignItems: 'center', pb: 6 }}>
         <Button variant='outlined' onClick={() => onGoBack()} disabled={messageInfo?.severity === 'info'} >Go back</Button>
-        <Button variant='contained' onClick={() => onCreateInstance()} disabled={messageInfo?.severity === 'info'}>Create instance</Button>
+        <Button id='ag_create_instance_btn' variant='contained' onClick={() => onCreateInstance()} disabled={messageInfo?.severity === 'info'}>Create instance</Button>
```
Comment (kiiamakir) [inline client/src/components/InstanceSummaryView.js:105]: Should separate these onto individual rows to make the line length a bit shorter

File: client/src/components/course-view/Assignments.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const Assignments = ({ attainments, formula, courseId, instance, handleAddPoints
       <Box sx={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', alignItems: 'center', gap: 1, mt: 2, mb: 1 }}>
         <Button onClick={() => navigate(`/${courseId}/create-attainment/${instance.id}`) }>Add attainment</Button>
         <Box sx={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'flex-start', alignItems: 'center', gap: 1 }}>
-          <Button variant='outlined' onClick={() => navigate(`/${courseId}/course-results/${instance.id}`)}>See course results</Button>
+          <Button id='ag_course_results_btn' variant='outlined' onClick={() => navigate(`/${courseId}/course-results/${instance.id}`)}>See course results</Button>
```
Comment (kiiamakir) [inline client/src/components/course-view/Assignments.js:48]: Here as well

**Review by kiiamakir — CHANGES_REQUESTED** [review]
Looks good! A few comments about line length

**Review by lakooi — APPROVED** [review]

**Review by kiiamakir — APPROVED** [review]

**Pull request merged**